eventsources/eventing.go

```diff
@@ -402,6 +467,23 @@ func (e *EventSourceAdaptor) run(ctx context.Context, servers map[apicommon.Even
 				}
 				if err = common.Connect(&backoff, func() error {
 					return s.StartListening(ctx, func(data []byte, opts ...eventsourcecommon.Options) error {
+						if filter, ok := filters[getFilterKey(s.GetEventSourceType(), s.GetEventName())]; ok {
+							dataMap := make(map[string]interface{})
+							err := json.Unmarshal(data, &dataMap)
+							if err != nil {
+								logger.Warn("unable to json unmarshal data")
```

Comment:
Something like `failed to unmarshal data, invalid json maybe?`, and print out the error message would be better.

Reply:
For this, would `logger.Errorw("failed to unmarshal data, invalid json", zap.Error(err)) work

Reply:
I would recommend to use `Warn` for this particular error.